user_events.py

```diff
@@ -535,6 +535,21 @@ def user_an_event_edit(id):
             update_user_event(id, post_by_id, {'endDate': ""})
         else:
             update_user_event(id, post_by_id, None)
+        if request.method == 'POST':
```

Comment:
The code in this if block is already under the `POST` condition. Please see line 243. Thanks.